docs/source/integrations/smart_tv/index.rst

```diff
@@ -0,0 +1,8 @@
+SMART TV's
```

Comment:
This needs to be added to `/docs/source/integrations/integrations.rst` so that it shows up on RTD.

Reply:
done